libs/barista-components/chart/src/range/range.spec.ts

```diff
@@ -41,6 +43,10 @@ import { RangeStateChangedEvent } from './range';
 
 // tslint:disable:no-magic-numbers no-unbound-method no-use-before-declare
 
+export class MockChart {
+  _focusTargets = new Set<DtChartFocusTarget>();
+}
+
 describe('DtChart Range', () => {
```

Comment:
I think it would make sense to add a test to check whether focus management works now as expected.

Reply:
As discussed in our 1on1 session, I added tests for shifting the focus on the anchor directive.